lib/api/src/externals/global.rs

```diff
@@ -208,6 +208,13 @@ impl Global {
     pub fn same(&self, other: &Self) -> bool {
         Arc::ptr_eq(&self.vm_global.from, &other.vm_global.from)
     }
+
+    /// Check if the global holds a strong `InstanceRef`.
+    /// None means there's no `InstanceRef`, strong or weak.
+    // TODO: maybe feature gate this, we only need it for tests...
```

Comment:
This is only used in tests. As such I would expect to only be visible there (can we delete it from here and move the logic somewhere else?)

Reply:
Unfortunately it can't be, these accessors access private info. It has to be a method on the type

Reply:
One solution to this would be to define a trait with these methods on it and export it in like `wasmer::test` and mark that module as `#[hidden]`so it doesn't show up in docs. Then these methods will only be available if you import `wasmer::test::SomeTrait`